### PR TITLE
Set ESPRESSO_SEQUENCER_IS_DA=true in docker demo

### PR DESCRIPTION
**Pull request by sveitser**
- In order for the `is_da` variable to be set to true the env var cannot be empty.
- Log the non-sensitive sequencer options on startup instead of only the modules. This should make it easier to spot configuration errors.
- Log with info level instead of warn since they aren't indications of anything potentially being wrong.

### CODE REVIEW

File: sequencer/src/main.rs [hunk 1]
```diff
@@ -18,10 +18,10 @@ async fn main() -> anyhow::Result<()> {
     setup_logging();
     setup_backtrace();
 
-    tracing::warn!("sequencer starting up");
```
Comment (sveitser) [inline sequencer/src/main.rs:21]: @jbearer just saw you recently changed this. Feels sort of wrong to log with warn or error level if there's no cause for alarm. Wondering why this was changed.

**Comment by rob-maron** [comment]
This is my fault, I incorrectly assumed it was pulling from the ".env", but I had removed it from there

**Review by rob-maron — APPROVED** [review]

File: sequencer/src/main.rs [hunk 2]
```diff
@@ -18,10 +18,10 @@ async fn main() -> anyhow::Result<()> {
...
+    tracing::info!("sequencer starting up");
     let opt = Options::parse();
+    tracing::info!("options: {:?}", opt);
```
Comment (jbearer) [inline sequencer/src/main.rs:23]: For this one I think `warn!` does make sense, as we would almost always want this to show up in the logs even if we are trying to hide other INFO level messages.

**Review by jbearer — APPROVED** [review]
Nice!

File: sequencer/src/main.rs (hunk 2 above)
Comment (sveitser) [reply sequencer/src/main.rs:23]: Ok I'll change it back

Comment (sveitser) [reply sequencer/src/main.rs]: https://github.com/EspressoSystems/espresso-sequencer/pull/1409/commits/69a8b5fd1619faa1d41802f07de8eb753ddc1634

**Pull request merged**